draft-pda-protocol.md

```diff
@@ -540,24 +549,20 @@ To encrypt an input share, the client first generates an HPKE
 {{!I-D.irtf-cfrg-hpke}} context for the aggregator by running
 
 ~~~
-enc, context = SetupBaseS(pk, [TODO])
+enc, context = SetupBaseS(pk, "pda input share" || server_role)
 ~~~
 
-where `pk` is the aggregator's public key. `enc` is the encapsulate HPKE
-context and `context` is the HPKE context used by the client for encryption.
-The payload is encrypted as
+where `pk` is the aggregator's public key and `server_role` is a byte whose
+value is `0x01` if the aggregator is the leader and `0x00` if the aggregator is
+the helper. `enc` is the encapsulate HPKE context and `context` is the HPKE
+context used by the client for encryption. The payload is encrypted as
 
 ~~~
-payload = context.Seal(input_share, [TODO])
+payload = context.Seal(input_share, task_id || time || jitter)
 ~~~
 
-where `input_share` is the aggregator's input share.
-
-[TODO: Fully specify encryption of the shares. We need to make sure we
-authenticate things like the PDA parameters and the report timestamp. We need to
-decide what the info string for SetupBaseS() will be, as well as the aad for
-context.Seal(). The aad might be the entire "transcript" between the client and
-aggregator.]
+where `input_share` is the aggregator's input share and `task_id`, `time`, and
+`jitter` are the fields of `PDAReport`.
```

Comment:
Should we say that leaders should drop reports whose timestamps fall outside of the batch window? (Happy for that to be an open issue, if desired.)

Reply:
I think this would be a bit confusing here. Ideally this is clear from {{pa-aggregate}}.